lib/installer.js

```diff
@@ -41,7 +41,7 @@ async function installWAD (msiPath) {
 }
 
 async function verifyWAD () {
-  log.info("Verifying WinAppDriver is installed with correct checksum");
+  log.info("Verifying WinAppDriver version " + WAD_VER + " is installed via comparing the checksum.  You must use WinAppDriver version " + WAD_VER);
```

Comment:
Our style decision is to use string interpolation instead of concatenation. So here and below:

```js
log.info(`...version ${WAD_VER} is installed...`);
```